docker/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
```

Comment:
I think this can be simplified to:

source conf.sh
docker build -t coscale/cli .